scm-ui/ui-components/src/repos/changesets/Changesets.stories.tsx

```diff
@@ -37,7 +37,7 @@ import { Changeset } from "@scm-manager/ui-types";
 import { Replacement } from "../../SplitAndReplace";
 
 const Wrapper = styled.div`
-  margin: 2rem;
+  margin: 4rem;
```

Comment:
Most of the signatures are not readable.
```suggestion
  margin: 25rem 4rem;
```